v22.2/with-storage-parameter.md

```diff
@@ -40,10 +40,6 @@ The `WITH (storage parameter)` [statement](sql-statements.html) sets a storage p
 
 {% include {{ page.version.version }}/misc/table-storage-parameters.md %}
 
-### Row-level TTL parameters
-
-For the list of storage parameters that affect how [Row-Level TTL](row-level-ttl.html) works, see the list of [TTL storage parameters](row-level-ttl.html#ttl-storage-parameters).
```

Comment:
Nit: Just curious about this removal. It seems to still point to valid info on that page.

Reply:
That block has been moved into the `table-storage-parameters.md` include just above, as (after discussing with Rich) I believe it's relevant everywhere that include is used.